src/main/java/teammates/common/datatransfer/InstructorBundle.java

```diff
@@ -0,0 +1,7 @@
+package teammates.common.datatransfer;
+
+/**
+ * Represents details of an instructor account.
+ */
+public class InstructorBundle extends CommonBundle {
```

Comment:
Per the previous comment, this could be named `InstructorAccountSearchResult`.

Reply:
Updated, see fe94266